server/sonar-server/src/main/java/org/sonar/server/setting/ws/SetAction.java

```diff
@@ -130,13 +129,11 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    DbSession dbSession = dbClient.openSession(false);
-    try {
+    userSession.checkLoggedIn();
```

Comment:
this is a change of behavior which is not necessary : "AnyOne" could be given permission admin on the project.